src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -75,7 +76,7 @@ class RelaxedMemberPermission(OrganizationPermission):
     scope_map = {
         "GET": ["member:read", "member:write", "member:admin"],
         "POST": ["member:write", "member:admin"],
-        "PUT": ["member:write", "member:admin"],
+        "PUT": ["member:write", "member:admin", "member:invite"],
```

Comment:
tiny nit: keep in order of access
```suggestion
        "PUT": ["member:invite", "member:write", "member:admin"],
```